apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/InpatientClaimTransformerV2Test.java

```diff
@@ -688,15 +688,21 @@ public void shouldHaveDiagnosesList() {
   @Test
   public void shouldHaveDiagnosesMembers() {
     DiagnosisComponent diag1 =
-        TransformerTestUtilsV2.findDiagnosisByCode("R4444", eob.getDiagnosis());
+        TransformerTestUtilsV2.findDiagnosisByCode("A05", eob.getDiagnosis());
 
     DiagnosisComponent cmp1 =
         TransformerTestUtilsV2.createDiagnosis(
             // Order doesn't matter
             diag1.getSequence(),
             List.of(
-                new Coding("http://hl7.org/fhir/sid/icd-10-cm", "R4444", null),
-                new Coding("http://hl7.org/fhir/sid/icd-10", "R4444", null)),
+                new Coding(
+                    "http://hl7.org/fhir/sid/icd-10-cm",
+                    "A05",
+                    "\"OTH BACTERIAL FOODBORNE INTOXICATIONS, NEC\""),
```

Comment:
Is this really a string that include the double-quotes?

Reply:
Just removed all weirdly encoded diagnoses.